rosbag2_transport/test/rosbag2_transport/test_record.cpp

```diff
@@ -80,31 +93,31 @@ TEST_F(RecordIntegrationTestFixture, qos_is_stored_in_metadata)
   auto recorded_topics = writer.get_topics();
   std::string serialized_profiles = recorded_topics.at(topic).offered_qos_profiles;
   // Basic smoke test that the profile was serialized into the metadata as a string.
+  EXPECT_THAT(serialized_profiles, ContainsRegex("reliability: 1\n"));
+  EXPECT_THAT(serialized_profiles, ContainsRegex("durability: 2\n"));
+  EXPECT_THAT(
+    serialized_profiles, ContainsRegex(
+      "deadline:\n"
+      "    sec: .+\n"
+      "    nsec: .+\n"
```

Comment:
string contains "deadline:"?

Reply:
i'm slightly interested in the rmw_time_t being present as well, but not in the specific values